docs/advanced-topics/custom-transports.md

```diff
@@ -3,21 +3,21 @@ id: custom-transports
 title: Custom Transports
 ---
 
-The MLAPI uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports. 
+Netcode for Gameobjects (Netcode) uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports. 
 
 A Transport collects messages from the application and transmits them over the network. 
 The Transport layer is also responsible for making sure that all packets arrive and that they arrive in order if needed. Transports also make sure that your data won't get corrupted when it's sent over the network.
 
 ## Community Contribution
 
-The MLAPI supports community contributions for Transport implementations. See the [MLAPI Community Contributions](https://github.com/Unity-Technologies/mlapi-community-contributions)  repository to learn more.
+Netcode supports community contributions for Transport implementations. See the [Netcode Community Contributions](https://github.com/Unity-Technologies/mlapi-community-contributions)  repository to learn more.
 
 To use a Transport, follow the [Community Transport Installation Guide](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports).
```

Comment:
This whole "Community Contributions" section might need a rework since we're going to keep that contribs repo more broad as Multiplayer, not just for the SDK. @LukeStampfli